### PR TITLE
MW-46: Remove Products with Zero-Quantity Values from Order Export File

### PR DESCRIPTION
**Pull request by jkondrat**

### CODE REVIEW

File: src/main/java/org/openlmis/fulfillment/service/OrderCsvHelper.java [hunk 1]
```diff
@@ -98,10 +102,14 @@ private void writeHeader(List<OrderFileColumn> orderFileColumns, Writer writer)
   private void writeLineItems(Order order, List<OrderLineItem> orderLineItems,
                               List<OrderFileColumn> orderFileColumns, Writer writer)
       throws IOException {
+    Boolean includeZeroQuantity = configurationSettingService.getBoolValue(
```
Comment (palbecki) [inline src/main/java/org/openlmis/fulfillment/service/OrderCsvHelper.java]: Can't we take it from environment variables?

File: demo-data/fulfillment.configuration_settings.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+[
+  {
```
Comment (palbecki) [inline demo-data/fulfillment.configuration_settings.json]: no need for this?

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -33,4 +33,6 @@ auth.resourceId=fulfillment
 referencedata.url=${BASE_URL}
 notification.url=${BASE_URL}
 
-email.noreply=${MAIL_ADDRESS}
\ No newline at end of file
+email.noreply=${MAIL_ADDRESS}
+
+order.export.includeZeroQuantity=${ORDER_EXPORT_INCLUDE_ZERO_QUANTITY:false}
```
Comment (palbecki) [inline src/main/resources/application.properties]: Can you update readme with description for that env variable? Or just submit PR for openlmis-config.

Comment (jkondrat) [inline src/main/resources/application.properties]: https://github.com/OpenLMIS/openlmis-config/pull/1

**Pull request merged**